sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -322,6 +323,68 @@ main().catch((err) => {
 
 _Note_: you may also use the `beginAnalyzeDocuments` method to extract generic document information using the prebuilt document model by providing the model ID `"prebuilt-document"`. This method provides a weaker TypeScript type for the layout analysis result, but will produce the same information. The `beginExtractGenericDocument` method is available for your convenience.
 
+### Read Document
+
+```javascript
+const { DocumentAnalysisClient, AzureKeyCredential } = require("@azure/ai-form-recognizer");
+
+async function main() {
+  const endpoint = "<cognitive services endpoint>";
+  const apiKey = "<api key>";
+  const path = "<path to a document>"; // pdf/jpeg/png/tiff formats
+
+  const readStream = fs.createReadStream(path);
+
+  const client = new DocumentAnalysisClient(endpoint, credential);
```

Comment:
should `credential` be an AzureKeyCredential with the apiKey passed in?

Reply:
Thanks, just caught the same error testing this fragment. I combined samples and that ended up disconnected.